README.md

```diff
@@ -1,22 +1,48 @@
 # Mainflux IoT Admin UI based on Angular 8+ and <a href="https://github.com/akveo/nebular">Nebular</a>
 
-## Install
-Build Docker image:
+## Installation
+
+### Quick Setup
+For a quick setup, pre-built images from Docker Hub can be used.
+
+First, make sure that `docker` and `docker-compose` are installed. Also, stop existing Mainflux containers if any.
+
+Then, use the following instructions:
 ```
-make ui
+$ git clone https://github.com/mainflux/ui.git
+$ cd ui
+$ make run
 ```
+UI should be now up and running at `http://localhost/`.
 
-Run docker-compose:
+*(Note that `http://localhost:3000/` is for internal use only, and is not intended to be used by the end-user.)*
+
+More configuration (port numbers, etc.) can be done by editing the `.env` file before `$ make run`.
+
+### Developer Setup
```

Comment:
Move this into `Usage` chapter, and just in one sentence explain that developer build can be achieved with this `make` commmand, but would be also goot to list a command that `make ui` executes.

Reply:
I'm a bit confused here. I renamed the chapter to `usage` and trimmed the sentences but maybe I didn't understand you correctly.

Reply:
This looks good.